rules/naming/src/ValueObject/PropertyRename.php

```diff
@@ -26,27 +26,27 @@ final class PropertyRename
     private $property;
 
     /**
-     * @var Type
+     * @var ClassLike
      */
-    private $propertyType;
```

Comment:
PropertyType was removed because it is unused at the moment. It will be used in `BreakingVariableRenameGuard::isRamseyUuidInterface` and `BreakingVariableRenameGuard::isDateTimeAtNamingConvention` once I'll get  Param renaming.
For reference https://github.com/rectorphp/rector/pull/4242#discussion_r489739962